tests/aggregation/models.py

```diff
@@ -30,6 +30,7 @@ class Book(models.Model):
     contact = models.ForeignKey(Author, models.CASCADE, related_name="book_contact_set")
     publisher = models.ForeignKey(Publisher, models.CASCADE)
     pubdate = models.DateField()
+    print_info = models.JSONField(null=True, default=None)
```

Comment:
@camuthig this is the source of all your Oracle test failures...

Django uses a `NCLOB` column to persists `JSONField` and Oracle doesn't allow to `GROUP BY` a `NCLOB` field.

I don't think the usage of a `JSONField` is mandatory to test this feature so I'd suggest using a different field type instead.